### ISSUE TITLE
03-wordemb-pytorch/wordemb-cbow.py : softmax missing ?

### ISSUE THREAD

**Issue by frederictamagnan** [neubig/nn4nlp2017-code#36]
Shouldn't a softmax be added to the model output?
